### DISCUSSION TITLE
Cannot vote on Optimism

### DISCUSSION THREAD

**Discussion by JakELips** [snapshot-labs/snapshot#3408] [General]
I have 239,9967 Voting Power. Delegated my tokens some months ago. I am still unable to vote at Snapshot. Tokens are not in any contracts. Free for trading. Anyone any idea why I cannot vote ?

**Comment by ChaituVR** [comment]
Hi, if you are delegating to someone, your voting power is taken by them, and you won't be able to vote. make sure to undelegate or delegate yourself before the next vote to vote on future proposals

**Reply by JakELips** [reply]
Thanks for clarifying

**Comment by Argentino110** [comment]
Hi! It is possible to undelegate votes?

**Reply by dzhamilrustamov** [reply]
отмените делегирование монет
